src/modules/commerce/index.ts

```diff
@@ -67,14 +67,10 @@ export class CommerceModule {
       return `${symbol}${0.0}`;
     }
 
-    const randValue = this.faker.datatype.number({ max: max, min: min });
+    const randValue = this.faker.number.int({ min, max });
```

Comment:
Maybe simplify this to:

```suggestion
    const randValue = this.faker.number.float({ min, max, precision: 10 ** -dec });
```

?